staramr/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# base image
+FROM ubuntu:bionic
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="staramr"
+LABEL software.version="0.5.1"
+LABEL description="staramr scans bacterial genome contigs for AMR genes and plasmids"
+LABEL website="https://github.com/phac-nml/staramr"
+LABEL license="https://github.com/phac-nml/staramr"
+
+# Maintainer
+MAINTAINER Rachael St. Jacques <rachael.stjacques@dgs.virginia.gov>
+
+# install dependencies using conda
+RUN apt-get update && apt-get install -y \
+ python3.6\
+ python3-pip\
+ ncbi-blast+\
+ git
+
+#get staramr
+RUN pip3 install staramr
```

Comment:
please change to `pip3 install staramr==0.5.1`. If the staramr developers release another version, and the docker image gets built again, this will cause the latest version on pypi to be downloaded/installed. For these docker-builds, we'd like the versions of programs to be as static as possible, so the user knows exactly what version is installed.